src/main/java/de/zalando/aruha/nakadi/service/subscription/state/ClosingState.java

```diff
@@ -50,6 +50,9 @@ public void onEnter() {
     }
 
     private void onTopologyChanged() {
+        if (topologyListener == null) {
```

Comment:
I think it is good to check parameters for nulls and even better if you can do something with these nulls (recreate, go other way etc). In this particular situation if we do not have this check we will have NPE latter on. 
What are the reasons to have it ?